docs/tempo/website/Troubleshooting/_index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Troubleshooting
+weight: 550
+---
+
+# Troubleshooting missing traces
+
+This topic helps with day zero operational issues that may come up when getting started with Tempo. It walks through debugging each part of the ingestion and query pipeline to drill down and diagnose issues. 
+
+## 1. Problem - I am unable to see any of my traces in Tempo
+
+### Potential causes
```

Comment:
```suggestion
>** Potential causes**
```

The reason I think we should take it a note is because a h2 followed by h3 looks odd when there is no normal content in between.